MobileCenter/MobileCenter/MSMobileCenter.m

```diff
@@ -183,10 +184,12 @@ - (void)start:(NSString *)appSecret withServices:(NSArray<Class> *)services {
     if (configured) {
 
       NSArray *sortedServices = [self sortServices:services];
+      NSMutableArray<NSString*>* servicesNames = [NSMutableArray arrayWithCapacity:sortedServices.count];
 
       for (Class service in sortedServices) {
-        [self startService:service];
+        [servicesNames addObject:[self startService:service].storageKey];
```

Comment:
You are using a right value here but I think we need to add another method to return service name and use that instead.
Looks weird if someone looks at this line because of using storage key for service name even though it contains service name. @TroubleMakerBen and @clpolet, what do you think?